src/platforms/php/guides/laravel/index.mdx

```diff
@@ -33,6 +35,32 @@ public function register()
 }
 ```
 
+### Alternative: Log Channel
```

Comment:
What are users missing out if they pick this versus the other? I'm not familiar with PHP but wondering why there are these two alternatives

Reply:
Just saw @stayallive already went through. LGTM, just curious here?

Reply:
The log channel has a lot more noise considering it also contains, well logs. And not everything dumped in logs should end up in Sentry so the "normal" installation only captures uncaptured exceptions.

Reply:
Would you say it's best we add the log installation on a separate page? Doesn't seem to be clear to users landing here what path to take and if we prefer they go with unhandled exceptions only, perhaps that's all we should keep here

Reply:
It is currently on a [separate page](https://docs.sentry.io/platforms/php/guides/laravel/usage/#log-channels) but @sykezz found that too hidden and suggested this. I'm not a 100% sure what is the best solution here. 

Having 2 instructions might be a little adding more confusion that not having it. For the index page especially since doing both means you get every exception duplicated which of course not something we want to have users do by accident.

Maybe it would be better to add a new section under [customization](https://docs.sentry.io/platforms/php/guides/laravel/usage/#customization) linking to the [log channels page](https://docs.sentry.io/platforms/php/guides/laravel/usage/#log-channels) with a little excerpt that that is also a option?

Reply:
One of the principles of this page is to keep it as short and simple as possible. The downside is that we're hiding some possibilities and complexities from the user, but ideally this can be introduced at the very end with "next steps". What we want to avoid is having someone load this page and 'decide to take this later because no time to read a wall of text. Not to say this PR gets this in that place but this is the principal we try to follow.

That said, I'll defer to @imatwawana on what's best to do here. Thank you for clarifying everything🙏

Reply:
Thanks everyone for all the inputs. The reason I see log channel as an alternative setup is because if you choose log channel, you'd have to remove the code from the Exception Handler. With log channel approach, you've the option to push info and warnings into Sentry as well. It's like a little hidden feature :smiley:
Currently log channel setup is under [Usage->Customization->Log Channels](https://docs.sentry.io/platforms/php/guides/laravel/usage/#log-channels) and I find it a bit hidden. Also, doesn't it sound more related to [configuration](https://docs.sentry.io/platforms/php/guides/laravel/configuration/) rather than [usage customization](https://docs.sentry.io/platforms/php/guides/laravel/usage/#customization)? :sweat_smile:

I believe someone familiar with Laravel would be able to understand what the two approaches means, and would be able to choose. But, I agree that installation step should be short and less complicated. So like @stayallive suggested, how about a little excerpt on the installation page with the link to log channel section?

Reply:
Thanks, everyone! I'm going to add another suggestion, what about adding the page here:
![2021-11-05_11-43-43](https://user-images.githubusercontent.com/76437239/140539399-1b55b4be-8a67-4590-861e-3cdccb8643ba.png)

Reply:
That is not a bad idea I think 👍

Reply:
@sykezz, is this a change you feel comfortable/have time to make? If not, I'll spin up a new PR to make the changes (unfortunately I can't pull your branch in to do it on the same PR).

Reply:
@imatwawana yeah that sounds good. I'll push the changes soon.

Reply:
@imatwawana Hi. I've had some trouble adding that page under there. I don't think the current structure allows me to do that. Any advice?

Reply:
As it turns out, I don't know how to make create that structure either. Sorry about that! For the sake of not dragging out this process any longer,  let's just stick with adding a little section in the Install part of the Getting Started page that points to the content where it currently exists. Does that make sense?

Reply:
@imatwawana Yup. How about a little note?

![image](https://user-images.githubusercontent.com/6332979/143680849-cbb6b5d1-d4b1-4c7d-ac15-f0712f7faf27.png)

Reply:
yep! looks good

Reply:
Pushed the changes and resolved conflicts :smile: